nautobot/core/testing/forms.py

```diff
@@ -18,6 +19,7 @@ def test_form_dynamic_model_choice_fields_query_params(self):
                     continue
                 with self.subTest(f"Assert {self.form_class.__name__}.{field_name} query_params are valid."):
                     query_params_fields = set(field_class.query_params.keys())
+                    query_params_fields = query_params_fields - set(NON_FILTER_QUERY_PARAMS)
```

Comment:
This addition is to keep the test from failing due to the presence of `exclude_m2m=True` as a query parameter.